index.md

```diff
@@ -353,7 +353,7 @@ print("\\\\\d{3}")  # Error on this line
 
 ### R0204: Redefined variable type {#R0204}
 
-This error occurs when the type of a variable changes inside a method or a 
+This error occurs when the type of a variable changes inside a method or a
 function. See the examples below.
 
 ```python
```

Comment:
you can see some examples of the includes in this PR:

https://github.com/pyta-uoft/website/pull/16/files